FOOD.php

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang = "be">
+    <head>
+        <meta charset = "utf-8">
+        <meta name = "description" content = "Minsk guide">
+        <meta name = "keywords" content = "Minsk, hotels, food, museums, sights, guide, tourist, visit, travel,
+                                           Минск, отели, еда, музеи, достопримечательности, гайд, турист, посетить, путешествие,
+                                           Мінск, гателі, ежа, музеі, славутасці, гайд, турыст, наведаць, падарожжа">
+        <meta name = "author" content = "Oles Novikov">
+        <meta name = "viewport" content = "width=device-width, initial-scale=1.0, maximum-scale=5.0">
+        <title>Мінск - кухня</title>
+        <link rel = "stylesheet" href = "css/header.css">
+        <link rel = "stylesheet" href = "css/food.css">
+        <link rel = "stylesheet" href = "css/footer.css">
+        <link rel = "shortcut icon" href = "img/favicon.ico" type = "image/x-icon">
+        <link href = "https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel = "stylesheet">
```

Comment:
В HTML не нужны пробелы вокруг равно
```suggestion
        <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
```

Reply:
Исправил